partiql-playground/README.md

```diff
@@ -37,6 +37,39 @@ node src/server.ts
 ```
 6. your browser go to `http://localhost:8000/`
 
+## Run via docker container
+
+1. Ensure `docker` is installed on your machine, by running the following:
+```bash
+docker --version
+# Example output
+Docker version 20.10.17, build 100c701
+```
+2. Build the package:
+```bash
+make build
+```
+3. Build the container:
+```bash
+make container-build
+```
+3. Run the container:
```

Comment:
Numbering is off here

Reply:
Yep, I'd exclude the `README.md` review in this PR since it'll get overwritten in the 3rd PR.

Reply:
Actually, that's not the case for the numbers; let me fix those.

Reply:
Fixed.